### PR TITLE
full support for Active Choice Plug-in

### PR DESCRIPTION
**Pull request by armyers**
The changes provide full support for the capabilities of the Active Choices plug-in.

### CODE REVIEW

File: jenkins_jobs_active_choice/active_choice.py [hunk 1]
```diff
@@ -13,29 +13,23 @@
...
-    ('visible-item-count', 'visibleItemCount', 1),
-    ('reference', 'referencedParameters', ''),
-    ('filterable', 'filterable', False),
+    ('fallback-sandbox', 'sandbox', 'False'),
+    ('fallback-classpath', 'classpath', 'False'),
 ]
 
 
+# use logging from JJB
+import logging
```
Comment (bgaifullin) [inline jenkins_jobs_active_choice/active_choice.py]: import should be at begin of file

**Comment by bgaifullin** [comment]
Thanks for PR, looks good

**Review by bgaifullin — CHANGES_REQUESTED** [review]

**Comment by bgaifullin** [comment]
there is unresolved conflicts

File: jenkins_jobs_active_choice/active_choice.py (hunk 1 above)
Comment (armyers) [reply jenkins_jobs_active_choice/active_choice.py]: Hi Bulat,

The import statement is now grouped with the others and the merge conflicts have been resolved.

Thanks!

- Allen

**Pull request merged**